# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../npd6/use_gitnow.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From ad66f241cb63e9f7dff4db9b3864c8105dbc6171 Mon Sep 17 00:00:00 2001
From: Sean Groarke <sgroarke@gmail.com>
Date: Fri, 25 Jul 2014 15:10:46 +0200
Subject: [PATCH] Critical change to Makefile for switch to git

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a12b145..5addc44 100644
--- a/Makefile
+++ b/Makefile
@@ -34,7 +34,7 @@ INSTALL_PREFIX=/usr
 MAN_PREFIX=/usr/share/man
 DEBIAN=debian/
 TARGZ=npd6-$(VERSION)
-DEV:= -D'BUILDREV="$(VERSION).$(shell svnversion -n .)"'
+DEV:= -D'BUILDREV="$(VERSION).$(shell git describe --always )"'
 
 all: $(SOURCES) $(EXECUTABLE)
 

